include/UIKit/UIApplication.h

```diff
@@ -262,15 +241,15 @@ UIKIT_EXPORT_CLASS
 @property (getter=isProximitySensingEnabled, nonatomic) BOOL proximitySensingEnabled STUB_PROPERTY;
 @property (nonatomic) BOOL applicationSupportsShakeToEdit STUB_PROPERTY;
 @property (nonatomic) NSInteger applicationIconBadgeNumber;
-@property (nonatomic) UIInterfaceOrientation statusBarOrientation;
+@property (nonatomic) UIInterfaceOrientation statusBarOrientation STUB_PROPERTY;
 @property (nonatomic, assign) id<UIApplicationDelegate> delegate;
-@property (nonatomic, copy) NSString* preferredContentSizeCategory;
-@property (nonatomic, getter=isStatusBarHidden) BOOL statusBarHidden;
-@property (nonatomic, readonly) CGRect statusBarFrame;
+@property (nonatomic, copy) NSString* preferredContentSizeCategory STUB_PROPERTY;
+@property (nonatomic, getter=isStatusBarHidden) BOOL statusBarHidden STUB_PROPERTY;
+@property (nonatomic, readonly) CGRect statusBarFrame STUB_PROPERTY;
```

Comment:
we don't have NOTINPLAN macros for headers.

---
In reply to: [110295252](https://github.com/Microsoft/WinObjC/pull/2445#discussion_r110295252) [](ancestors = 110295252)